app/src/main/java/com/example/returnpals/composetools/ConfirmPickup.kt

```diff
@@ -52,59 +53,104 @@ class ConfirmPickup {
-
-                //Confirm Date
-                Text(day.toString()+" "+month.toString()+" "+date.toString(),Modifier,fontSize = 26.sp,maxLines = 1)
-                //Confirm type of pickup as hand off or leave on door step
-                Text(typeOfPickup)
-                //Avoid printing null if no address given
-                if (pickUpAddress.getAddressLine(0) != null ) {
-                    Text(text = pickUpAddress.getAddressLine(0))
-                }
-                if (pickUpAddress.getAddressLine(1) != null ) {
-                    Text(text = pickUpAddress.getAddressLine(1))
-                }
-                if (pickUpAddress.getAddressLine(2) != null ) {
-                    Text(text = pickUpAddress.getAddressLine(2))
-                }
-                //Confirm packages
-                Text("Packages", fontWeight = FontWeight.Bold,fontSize = 22.sp, color = Color.Black)
-                //Row with Icon and text
-                Row(horizontalArrangement =  Arrangement.Start, verticalAlignment = Alignment.CenterVertically) {
-                    IconManager().getBoxIcon(modifier = Modifier.height(22.dp))
-                    Text(numberOfDigital.toString()+" Package with digital label",fontSize = 20.sp,maxLines = 1)
-                }
-                //Row with Icon and text
-                Row(horizontalArrangement =  Arrangement.Start, verticalAlignment = Alignment.CenterVertically) {
-                    IconManager().getBoxIcon(modifier = Modifier.height(22.dp))
-                    Text(numberOfPhysical.toString()+" Package with physical label",fontSize = 20.sp,maxLines = 1)
-                }
-                //Print payment if due
-                //Should be able to skip if user is has already payed for a plan.
-                if (priceArray[0]!=0) {
-                    Text("Visa ending "+visaLastFour)
-                    Text("One-Time Return "+priceArray[0])
-                    Text("Tax "+priceArray[1])
-                    Text("Total "+priceArray[2])
-                }
-
-            }
+            whiteBox(
+                pickUpDate = pickUpDate,
+                typeOfPickup = typeOfPickup,
+                pickUpAddress = pickUpAddress,
+                numberOfDigital = numberOfDigital,
+                numberOfPhysical = numberOfPhysical,
+                priceArray = priceArray,
+                visaLastFour = visaLastFour
+            )
             //end of the white box summary
 
             //Navigation buttons
-            Row(Modifier) {
+            Row(horizontalArrangement = Arrangement.SpaceAround, modifier =  Modifier.fillMaxWidth()) {
                 ButtonManager.BackButton(onClick = backButton)
                 ButtonManager.NextButton(onClick = nextButton)
             }
         }
     }
+    @Composable
+    fun whiteBox(pickUpDate:Calendar,typeOfPickup:String,pickUpAddress:Address,numberOfDigital:Int,numberOfPhysical:Int,priceArray:Array<Int>,visaLastFour:Int) {
```

Comment:
I know its a normal convention in programming languages for function names to start with a lower case letter but whenever I do this with composables I get a warning: "Composable functions that return Unit should start with an uppercase letter." Also, every composable I've called from `andriodx.compose` starts with an uppercase letter. I'd prefer we followed what the standard is and started our composable functions (that don't return anything) with an uppercase letter. It's not a huge deal but something that bugs me.

Reply:
I like your suggestions. I think we should have a meeting as a group and decide on a coding style policy to follow. After a meeting is scheduled I'll create an issue to correct and change my code to that policy.  I'll close this conversation after I open the issue.